### ISSUE TITLE
sphinx-3: download sphinx 3 from either … there is no OR part

### ISSUE THREAD

**Issue by dilyanpalauzov** [jsuto/piler#63]
https://www.mailpiler.org/sphinx-3/ says:

> Get sphinx 3.3.1 from either piler download site (it contains only the binaries) https://download.mailpiler.com/generic-local/sphinx-3.3.1-bin.tar.gz Make sure the binaries are in /usr/bin directory.

After EITHER there is no OR.

**Comment by jsuto** [comment]
Is this a grammar bug only?

**Comment by dilyanpalauzov** [comment]
I do not know.  Are there alternative download locations?

**Comment by jsuto** [comment]
Yes, the sphinxsearch official site. Now I get it, I'll fix it soon, thanks for mentioning.

**Comment by dilyanpalauzov** [comment]
Now https://www.mailpiler.org/sphinx-3/ shows `https://sphinxsearch.com/downloads/current/` but in fact this is a hyperlink to `https://www.mailpiler.org/sphinx-3/Sphinxsearch%20site`, which does not work.

**Comment by dilyanpalauzov** [comment]
Can I use sphinx 3.7, or I have really to stick with version 3.3.1?

**Comment by jsuto** [comment]
You can. However, you may even use manticore which is a drop-in replacement.
